adafruit_lsm303dlh_mag.py

```diff
@@ -216,20 +221,22 @@ def mag_rate(self, value):
         reg_m = ((value & 0x07) << 2) & 0xFF
         self._write_u8(self._mag_device, _REG_MAG_CRA_REG_M, reg_m)
 
-    def _read_u8(self, device, address):
+    def _read_u8(self, device: I2CDevice, address: int) -> None:
```

Comment:
This returns `int` since it returns `self._BUFFER[0]`.

Reply:
See https://github.com/adafruit/Adafruit_CircuitPython_LSM303DLH_Mag/pull/20/commits/59f705c05b49d26224a5380b3aa214cab2749403